### ISSUE TITLE
Mipmaps should be generated by default

### ISSUE THREAD

**Issue by DrSakuu** [ruyo/VRM4U#261]
By default, mipmap generation is off in the VRM import settings, but this causes more problems than it fixes with matcaps. When mipmaps are off and VRAM is full, textures become corrupted in renders. Mipmaps are essential for correct operation, and other settings should be adjusted to achieve a higher quality. That's why mipmap generation should be on by default, because most users need them. If mipmaps cause problems with matcaps, the program should automatically disable them only for matcaps and not other textures.

**Comment by ruyo** [comment]
Yes, you are right, I think it would look good to generate Mipmap except for MatCap.
However, everything is loaded as NoMipmap when using runtime load.
I have concerns about the editor import and runtime load looking different.

I will see if there is a better way.
Thanks.
